### ISSUE TITLE
Accessibility guidance: Download accessibility posters

### ISSUE THREAD

**Issue by henocookie** [nhsuk/nhsuk-service-manual-community-backlog#405]
Use this issue to discuss the [Download accessibility posters](https://service-manual.nhs.uk/accessibility/download-accessibility-posters) page in the service manual.

We're particularly interested in hearing about how the posters have helped teams and individuals research, design and develop with accessibility as a focus.

**Comment by EllenDoyle** [comment]
@danjohnstonUX  did Amber complete this?

**Comment by sarawilcox** [comment]
Yes, @EllenDoyle and @danjohnstonUX. We published the changes. All done. But we need to keep this issue open because the [downloads page in the accessibility guidance](https://service-manual.nhs.uk/accessibility/download-accessibility-posters) links to it and calls for people to add any further feedback to this GitHub issue.

**Comment by pearsori** [comment]
I just checked those posters on an accessibility checker and they fail miserably on 19 counts - hardly a great advert
